app/models/task.rb

```diff
@@ -1,3 +1,14 @@
 class Task < ActiveRecord::Base
+  belongs_to :list
```

Comment:
If my list is destroyed, what happens to my tasks? Can I access them?

Reply:
Since there is no relational callback to destroy the tasks after the list is destroyed, I would think that you can still access them. I would need to implement an after_destroy callback on tasks as well as a dependent: :destroy on lists.